quipucords/api/scanjob/model.py

```diff
@@ -576,9 +546,8 @@ def complete(self):
         self.log_current_status()
 
     def fail(self, message):
-        """Fail a job.
+        """Change status from RUNNING to COMPLETE.
```

Comment:
shouldn't this be from RUNNING to FAILED.?

Reply:
Good catch! Fixed and pushed.